x-pack/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/advanced_section/components/annotations/annotations_switch.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useState, useContext, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCallOut, EuiSpacer, EuiSwitch } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { JobCreatorContext } from '../../../../../job_creator_context';
+import { Description } from './description';
+
+export const AnnotationsSwitch: FC = () => {
+  const { jobCreator, jobCreatorUpdate, jobCreatorUpdated } = useContext(JobCreatorContext);
+  const [annotationsEnabled, setAnnotationsEnabled] = useState(
+    jobCreator.modelPlot ?? jobCreator.annotations
+  );
+  const [showCallOut, setShowCallout] = useState(jobCreator.modelPlot && annotationsEnabled);
+
+  useEffect(() => {
+    if (jobCreator.modelPlot === undefined) {
+      jobCreator.modelPlot = false;
+    }
+
+    jobCreator.annotations = annotationsEnabled;
+    jobCreatorUpdate();
+  }, [annotationsEnabled]);
+
+  useEffect(() => {
+    setShowCallout(jobCreator.modelPlot === true && annotationsEnabled === false);
+  }, [jobCreatorUpdated, annotationsEnabled]);
+
+  function toggleAnnotations() {
+    setAnnotationsEnabled(!annotationsEnabled);
+  }
+
+  return (
+    <>
+      <Description>
+        <EuiSwitch
+          name="switch"
+          checked={annotationsEnabled}
+          onChange={toggleAnnotations}
+          data-test-subj="mlJobWizardSwitchAnnotations"
+          label={i18n.translate(
+            'xpack.ml.newJob.wizard.jobDetailsStep.advancedSection.enableModelPlotAnnotations.title',
+            {
+              defaultMessage: 'Enable model plot annotations',
```

Comment:
As suggested by @droberts195, this should be edited to `Enable model change annotations`

Reply:
Updated in https://github.com/elastic/kibana/pull/70678/commits/1b54be9b39e990830c85f2506d040e803725dad5